src/sentry/static/sentry/app/components/alert.jsx

```diff
@@ -8,7 +8,7 @@ import InlineSvg from 'app/components/inlineSvg';
 import space from 'app/styles/space';
 
 const StyledInlineSvg = styled(InlineSvg)`
-  margin-right: 12px;
+  margin-right: calc(${p => p.size} / 2);
```

Comment:
nice

Reply:
Do we need `calc` since it's JS? why not just evaluate it in JS?

Reply:
yeah good catch billy.

Reply:
I originally was doing this, but the size comes in as a string, not number. So it may be units other than pixels, which is why I made it a client side `calc`.